docs/fundamentals/app-host-overview.md

```diff
@@ -486,7 +486,7 @@ In the preceding code:
 This logic can easily be inverted to connect to an existing Redis resource when you're running locally, and create a new Redis resource when you're publishing.
 
 > [!IMPORTANT]
-> .NET Aspire provides common APIs to control the modality of resource builders, allowing resources to behave differently based on the execution mode. The fluent APIs are prefixed with `RunAs*` and `PublishAs*`. The `RunAs*` APIs influence the local development (or run mode) behavior, whereas the `PublishAs*` APIs influence the publishing of the resource.
+> .NET Aspire provides common APIs to control the modality of resource builders, allowing resources to behave differently based on the execution mode. The fluent APIs are prefixed with `RunAs*` and `PublishAs*`. The `RunAs*` APIs influence the local development (or run mode) behavior, whereas the `PublishAs*` APIs influence the publishing of the resource. For more information on how Azure resources use these APIs, see [App host run modes](../azure/integrations-overview.md#app-host-run-modes).
```

Comment:
```suggestion
> .NET Aspire provides common APIs to control the modality of resource builders, allowing resources to behave differently based on the execution mode. The fluent APIs are prefixed with `RunAs*` and `PublishAs*`. The `RunAs*` APIs influence the local development (or run mode) behavior, whereas the `PublishAs*` APIs influence the publishing of the resource. For more information on how Azure resources use these APIs, see [Azure integrations app host run modes](../azure/integrations-overview.md#app-host-run-modes).
```

It is surprising to use "App host run modes" on this link when it is specific to Azure.